tests/test_tools.py

```diff
@@ -78,22 +74,21 @@ def test_parse_mapping_as():
     assert parse_obj_as(Dict[int, int], inputs) == {1: 2}
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_schema():
     assert schema_of(Union[int, str], title='IntOrStr') == {
         'title': 'IntOrStr',
         'anyOf': [{'type': 'integer'}, {'type': 'string'}],
     }
     assert schema_json_of(Union[int, str], title='IntOrStr', indent=2) == (
         '{\n'
-        '  "title": "IntOrStr",\n'
         '  "anyOf": [\n'
         '    {\n'
         '      "type": "integer"\n'
         '    },\n'
         '    {\n'
         '      "type": "string"\n'
         '    }\n'
-        '  ]\n'
```

Comment:
Are these sorts of strict outputs deterministic? If they’re not, we probably should have some way to at least actually enable it, similar to the sort_keys option in json dumps

Reply:
Yes, just the order changed due to changes in how the schema is generated in v2 (everything is added as dict keys, so order is preserved). I think there are some minor improvements we _could_ do here (even if just taking more care to get a field ordering we like in GenerateJsonSchema), but I think it's worth ignoring for now.